apps/src/templates/ProjectTemplateWorkspaceIconV2.tsx

```diff
@@ -55,8 +57,13 @@ const ProjectTemplateWorkspaceIconV2: React.FunctionComponent<
       }}
       tooltipOverlayClassName={className}
     >
-      {/* FontAwesomeV6Icon does not work with WithTooltip. */}
-      <i className={'fa-kit fa-connected-level'} />
+      <button type="button" className={styles.iconButton}>
```

Comment:
We are wrapping it in a `button` to make it tabbable right? Is it worth adding a comment explaining that?

Reply:
Correct. Sure - I can add a comment.